api/tests/unit/domain/models/Organization_test.js

```diff
@@ -300,4 +300,32 @@ describe('Unit | Domain | Models | Organization', () => {
       });
     });
   });
+
+  describe('get#isScoAngManagingStudents', () => {
+
+    it('should return true when organization is of type SCO and is managing student', () => {
+      // given
+      const organization = domainBuilder.buildOrganization({ type: 'SCO', isManagingStudents: true });
+
+      // when / then
+      expect(organization.isScoAndManagingStudents).is.true;
+    });
+
+    it('should return false when organization is not of type SCO', () => {
+      // given
+      const organization = domainBuilder.buildOrganization({ type: 'SUP' });
+
+      // when / then
+      expect(organization.isScoAndManagingStudents).is.false;
+    });
+
+    it('should return false when organization is not managingStudent', () => {
```

Comment:
Commentaire concernant les deux cas SCO : Est-ce que ce serait mieux de les réunir dans un context SCO avec deux It pour managingStudent true/false ?

Reply:
Hmmm pour ce `it` là on pourrait mettre autre chose que du `SCO`. L'important est que isManagingStudent soit à `false`. Du coup, je pense que les mettre dans un context "sco" peut apporter une info pas forcément nécessaire ici 🤔